### DISCUSSION TITLE
[Large Taskbar Icons] Visual enhacement for opening apps

### DISCUSSION THREAD

**Discussion by mateusfn98** [ramensoftware/windhawk-mods#129] [Q&A] [answered]
Can someone add some new visual enhacement to see the apps that are running, like in w10:
![image](https://user-images.githubusercontent.com/74764525/224388559-7ffb3d0f-b126-495b-83c6-71d60a700374.png)

In w11 it's awful, and when resizing the little dot just disappear.
![image](https://user-images.githubusercontent.com/74764525/224388833-eb2088d8-36b2-4d0b-9523-0db50b534dcd.png)

**Comment by m417z** [comment, marked as answer]
I added it to settings in version 1.1 of the Taskbar Labels for Windows 11 mod. More details about the new version:
https://ramensoftware.com/windhawk-mod-update-taskbar-labels-for-windows-11

![image](https://user-images.githubusercontent.com/4129781/224510098-cb3be880-7ebd-4784-af0c-e231174a085d.png)

If you prefer not to have labels, you can just set the width to the default value, 44, and configure the running indicator style:

![image](https://user-images.githubusercontent.com/4129781/224510100-d3baa80f-c257-45c8-a4ff-31490f4eb9c1.png)

**Reply by mateusfn98** [reply]
thanks, that's useful.
but some browsers and others apps, that bar don't appear or are out of the vision.
it's possible to set a variable to the vertical space between icon and indicator ?

**Reply by m417z** [reply]
> that bar don't appear or are out of the vision

What do you mean? Can you post a screenshot?

**Reply by mateusfn98** [reply]
relax, it's a monitor problem, i have some deadpixel in the bottom and the bar it's in this area, don't worry

**Reply by m417z** [reply]
That's a quite specific problem. You can fork the mod and change it to your needs. For changing the indicator position, you can try playing with the indicator margins [here](https://github.com/ramensoftware/windhawk-mods/blob/944eed5252e88f92ec24a738af705c092f3ac0c4/mods/taskbar-labels.wh.cpp#L663).
